Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public static class Lesson4
+    {
+        public static void Demo()
+        {
+            WelcomePrompt();
+            do
+            {
+                PromptUserForStats();
+
+            } while (PromptCalculateNewBMI());
+        }
+        public static bool PromptCalculateNewBMI()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'PromptCalculateNewBMI' to match pascal case naming rules, consider using 'PromptCalculateNewBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884599)